include/wx/localedefs.h

```diff
@@ -108,6 +108,28 @@ struct WXDLLIMPEXP_BASE wxLanguageInfo
     const char* TrySetLocale() const;
 };
 
+// ----------------------------------------------------------------------------
+// wxLocaleIdent: contains platform dependent language name
```

Comment:
This comment is not really correct any more, I'd say:
```suggestion
// wxLocaleIdent: allows to fully identify a locale under all platforms
```